src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/producer/MessageProducerImpl.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.producer;
+
+import org.apache.avro.Schema;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.common.ApplicationLogger;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.model.ChipsKafkaMessage;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.service.ServiceException;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+@Component
+public class MessageProducerImpl implements MessageProducer {
+
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm:ss");
```

Comment:
`formatter` is a constant and should be all uppercase.

Reply:
Removed - no longer needed.